Dockerfile

```diff
@@ -53,25 +52,37 @@ RUN echo "Downloading RapidPro ${RAPIDPRO_VERSION} from https://github.com/$RAPI
     tar -xf rapidpro.tar.gz --strip-components=1 && \
     rm rapidpro.tar.gz
 
+# Install Rust, it's required to build poetry on this version of alpine
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+# Install poetry
+RUN pip install -U pip && pip install -U poetry
+
+# Uninstall Rust
+RUN rustup self uninstall -y
```

Comment:
Without putting all of these in the same `RUN`, uninstalling will actually increase the image size, instead of decreasing it, due to the way the docker storage layering works.

Reply:
ah ok. I'll combine it all